file_store.go

```diff
@@ -29,6 +29,9 @@ import (
 // to keep the credentials in plain-text.
 //
 // Reference: https://docs.docker.com/engine/reference/commandline/cli/#docker-cli-configuration-file-configjson-properties
+//
+// Deprecated: This type is deprecated.
+// The same functionality is now provided by oras-go.
```

Comment:
Instead of just mentioning `oras-go`, it is better to mention the detailed package (e.g. `oras.land/oras-go/v2/registry/remote/credentials`).

Reply:
Mentioned it in the package description.

Reply:
Do we need to have the link in everywhere?

Reply:
But it would look a bit strange: 
![image](https://github.com/oras-project/oras-credentials-go/assets/8113721/2a3f3fe9-0d19-4667-ae60-4afa44e7903d)

Reply:
Updated.